### PR TITLE
add coverage badge

### PR DESCRIPTION
**Pull request by lanwen**
/cc @oleg-nenashev @KostyaSha 

[Review on Reviewable](https://reviewable.io/reviews/jenkinsci/github-plugin/92)

### CODE REVIEW

**Comment by KostyaSha** [comment]
We shouldn't show such numbers to people :D

**Comment by lanwen** [comment]
62% not bad :) i have plan to encrease it up to 80%

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 Jenkins Github Plugin
 =====================
+[![covarage](https://img.shields.io/sonar/http/sonar.lanwen.ru/com.coravy.hudson.plugins.github:github/coverage.svg?style=flat)](http://sonar.lanwen.ru/dashboard/index?id=com.coravy.hudson.plugins.github:github)
```
Comment (oleg-nenashev) [inline README.md]: "covarage"

Comment (KostyaSha) [inline README.md]: when it hit 80% it will be "coverage" :D

Comment (lanwen) [inline README.md]: oops

**Comment by KostyaSha** [comment]
:+1:

**Pull request closed without merging**